runtests.sh

```diff
@@ -91,19 +97,19 @@ if [[ $TARGET == *-darwin ]]; then
 fi
 
 # Check if we need to setup the network
-ip link show br-mullvadtest >&/dev/null || sudo ./scripts/setup-network.sh
+ip link show br-mullvadtest >&/dev/null || ./scripts/setup-network.sh
```

Comment:
There's an issue here: `dnsmasq` isn't killed when this is run inside temporary namespaces.